ui/config/setupTests.js

```diff
@@ -21,4 +21,12 @@ console.error = jest.fn(error => { throw new Error(error); }); //eslint-disable-
 console.warn = jest.fn(warn => { throw new Error(warn); }); //eslint-disable-line no-console
 
 // flag for GenericLoader
-global.GENERIC_LOADER_THROW_ON_REJECT_IN_TEST = true;
\ No newline at end of file
+global.GENERIC_LOADER_THROW_ON_REJECT_IN_TEST = true;
+
+// Unhandled promise rejections should fail tests
```

Comment:
This makes tests stricter - it fails if there's an unhandled promise rejection (which in practice happens when a component makes a request to the server that's not mocked).